mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -22,7 +24,7 @@ impl Settings {
         let mut config = Config::new();
 
         config.merge(File::from_str(DEFAULTS, FileFormat::Json))?;
-        config.merge(Environment::with_prefix(ENVIRONMENT_PREFIX))?;
+        config.merge(Environment::new())?;
```

Comment:
could you please get more details on this?

Reply:
it used to filter for env variables starting with IOTEDGE, but then I added a new env to enable/disable bridge which doesn't start with the prefix so changed this to read all env variables